profile/README.md

```diff
@@ -1,31 +1,17 @@
-## Hi there 👋
+![OpenSCD Header Picture](./assets/header.png)
+# ⚡️ OpenSCD - IEC 61850 SCL editing tool​
 
-🙋‍♀️ OpenSCD product vision
+**For:** utility/integrator/vendor companies​  
+**Who want:** to have flexibility, agility and efficiency in their SCL configurations​  
+**That offers:** a web front-end, open source, extensible, customizable, user friendly, SCL editing​  
+**Unlike:** proprietary and vendor specific tools​
```

Comment:
I prefer to avoid pejoratives, I would remove the "Unlike" and retain a positive vision. TBH I think there are good things about proprietary and vendor specific tools, even if they are not my preference...

Reply:
So, you would remove it completely?

Reply:
It's just me, I don't feel strongly about it :wink:

Reply:
I'm neutral. I understand both sides. Maybe we can discuss it in the team

Reply:
Let's remove it